src/app/api/workflow-states/workflowStates.service.ts

```diff
@@ -11,14 +11,33 @@ class WorkflowStatesService extends BaseService {
     const policyGate = new PoliciesService(this.user)
     policyGate.authorize(UserAction.Read, Resource.Tasks)
 
-    const filters = { where: { workspaceId: this.user.workspaceId } }
-    let workflowStates = await this.db.workflowState.findMany(filters)
+    let workflowStates: WorkflowStateResponse[] = await this.db.$queryRaw`
+    SELECT * FROM "WorkflowStates"
```

Comment:
Type will be `WorkflowState[]` instead, where that type comes from the schema defined in `@prisma/client`